specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/ServiceFabricManagedClusters/preview/2021-01-01-preview/managedapplication.json

```diff
@@ -1415,7 +1415,7 @@
           "description": "AveragePartitionLoadScalingTrigger"
         }
       ],
-      "x-ms-discriminator-value": "AveragePartitionLoadTrigger",
+      "x-ms-discriminator-value": "AveragePartitionLoad",
```

Comment:
what is causing this change?

Reply:
To fix the Swagger SemanticValidation error: `The value of x-ms-discriminator-value is not in the discriminator enum list: AveragePartitionLoadTrigger.`
However, this issue only exists in the older API versions and not in the new ones. I have already reverted the related changes in the current PR. How do you suggest we proceed?